src/pages/sign/signUp/page.tsx

```diff
@@ -1,11 +1,169 @@
-import { Layout } from "antd";
+/* eslint-disable react/jsx-props-no-spreading */
+import { Form, Input, Layout, Button } from "antd";
+import { useUnit } from "effector-react";
+import { routes } from "../../../shared/lib/atomic-router/route";
 import { PageHeader } from "../../../shared/ui/PageHeader/PageHeader";
+import { $users, addUser } from "./model";
+import "./page.css";
 
 const { Content } = Layout;
 
-export const SignUpPage = () => (
-  <Layout>
-    <PageHeader title="Регистрация" />
-    <Content />
-  </Layout>
-);
+type User = {
+  nickName: string;
+  name: string;
+  surname: string;
+  patronymic: string;
+  password: string;
+  confirm: never;
+};
+
+export const SignUpPage = () => {
+  const [users, addUserFn] = useUnit([$users, addUser]);
+
+  const onFinish = (values: User) => {
+    const user = { ...values, isAdmin: false };
+    delete user.confirm;
+    addUserFn(user);
+    // eslint-disable-next-line effector/mandatory-scope-binding
```

Comment:
dont disable lines! its very bad practise

Reply:
in last commit it was deleted